query/common/src/main/java/org/codice/ddf/admin/common/services/ServiceCommons.java

```diff
@@ -88,7 +88,8 @@ public String resolveProperty(String str) {
     Configurator configurator = configuratorSuite.getConfiguratorFactory().getConfigurator();
     configurator.add(configuratorSuite.getServiceActions().build(pid, newConfig, true));
 
-    // TODO RAP 13 Jul 17: Blank out password in the parameters passed here
+    // TODO RAP 13 Jul 17: Blank out password in the parameters passed here. Remove sonar
```

Comment:
Same question

Reply:
This TODO is still valid. I'm gonna go ahead and create a ticket for it.